keyboards/4by3/rules.mk

```diff
@@ -0,0 +1,11 @@
+MCU 				= atmega32u4
+F_CPU 				= 16000000
```

Comment:
As of #6253, `F_CPU`, `F_USB`, `ARCH`, and `OPT_DEFS` can actually be omitted for the ATmega32U4 if they are left at default :)

Reply:
Removed `F_CPU`, `F_USB`, `ARCH`, and `OPT_DEFS`